homeassistant/components/switch/rachio.py

```diff
@@ -0,0 +1,225 @@
+import logging
+import voluptuous as vol
+from datetime import timedelta
+
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util as util
+from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+REQUIREMENTS = ['https://github.com/Klikini/rachiopy'
```

Comment:
If this fork is intended to stay as a permanent code, could you push it to pypi?